packages/react/package.json

```diff
@@ -35,7 +35,8 @@
     "type-check": "tsc"
   },
   "dependencies": {
-    "hig-vanilla": "^0.1.26"
+    "hig-vanilla": "^0.1.26",
+    "react-transition-group": "^2.2.1"
```

Comment:
Did you manage to get an example working with several banners animating? I might suggest `react-flip-move` for an easier time with the animations (https://github.com/Autodesk/hig/pull/797).

Reply:
There should only be one banner on a screen at any given time. Either way, I don't plan on using a lib for this.